hugo/layouts/news-and-press/single.html

```diff
@@ -0,0 +1,76 @@
+{{ $projects := $.Site.Data.projects }}
+
+{{ $headCtx := dict "ctx" . }}
+{{ partial "head.html" $headCtx }}
+
+<body>
+{{ partial "header.html" }}
+<div class="text-left pb-5">
+    <br><br>
+    <h1 class="title text-left p-0 mt-5 mb-4">News, Press & Analyst Relations at source{d}</h1>
+    <h2 class="text-left h4 text-light mb-5">Explore the latest news, press coverage and analyst reports about source{d}</h2>
+</div>
+</div>
+</header>
+
+<section id="releases" class="text-dark pb-5">
+    <div class="container pt-5">
+        <div class="row">
+            <script src='http://www.globenewswire.com/Scripts/gnw_widget.js' type='text/javascript'></script> <script type='text/javascript'> var userActivityUrl ='https://www.globenewswire.com/SocialMediaActivity/Log/jsWidgetRefreshed/50522'; GetWidget('http://www.globenewswire.com/JsonData/organization/9HOlujazceMq1zUGLRgBfA%3d%3d?callback=?', 'Press Releases'); </script><div class='widget__width-height--container'> <div id='gnw_widget' class='gnw_content_container'></div></div> <div class='hideme'></div><div id='hidePbDate' class='hidePbDate'></div>
```

Comment:
I think there's a problem with Mixed Content.
I think you should review how _globenswire_ works, and how to load its content from an `HTTPS` site.
![image](https://user-images.githubusercontent.com/2437584/58325171-42bdba80-7e29-11e9-89c8-c5b4400aede2.png)

Reply:
Hi @dpordomingo,

I just used the javascript option from the site. It seems there are also RSS and ATOM options:

![image](https://user-images.githubusercontent.com/14981468/58339995-1450d700-7e4b-11e9-88ce-85ce387d7107.png)

If we need to use an integration that is not straight forward as the javascript maybe I will need a bit of help.

Thank you!

Reply:
I think you could contact their support to check if there is an option to load their JS from a secure page. 
Or you could also try changing the url of the script to use `https`